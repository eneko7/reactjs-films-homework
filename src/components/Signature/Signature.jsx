import React from 'react';
import style from './Signature.scss';

class Signature extends React.Component {
    render () {
        return (
            <span className={style.main_container_footer_copyright_signature}>{this.props.name}</span>
        );
    }
}

export default Signature;
