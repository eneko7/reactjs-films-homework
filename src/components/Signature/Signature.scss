.main_container_footer_copyright_signature{
    font-size: 20px;
    color: #969797;
}