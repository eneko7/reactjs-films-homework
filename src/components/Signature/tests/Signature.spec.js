import React from 'react';
import renderer from 'react-test-renderer';

import Signature from '../index';

describe('Signature', () => {
    it('Signature renders correctly', () => {
        const renderedSignature = renderer.create(<Signature name="IHAR KARPUK"/>).toJSON();
        expect(renderedSignature).toMatchSnapshot();
    });
});