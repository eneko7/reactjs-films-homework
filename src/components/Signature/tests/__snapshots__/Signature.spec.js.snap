// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signature Signature renders correctly 1`] = `
<span
  className="main_container_footer_copyright_signature"
>
  IHAR KARPUK
</span>
`;
